spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/DataFlowFlywayConfigurationCustomizer.java

```diff
@@ -34,6 +35,7 @@
  * active db vendor.
  *
  * @author Janne Valkealahti
+ * @author Chris Bono
```

Comment:
Feels weird putting my name here - but went ahead and did since I am resurrecting the files for mysql57 and figured it would be an easy signal who to ping when there are issues w/ it etc..

Reply:
We can't really fix this until boot differentiates between mysql/maria afaik. This means `mysql` package would become `mariadb` and then this stuff would need to go under `mysql` package.

Reply:
We would need to rename the directories at that point but this approach seems to work until then.

Reply:
W/ the Boot 2.7 upgrade - this goes away.